src/Catalyst.Core.Lib/Kernel/Kernel.cs

```diff
@@ -114,6 +116,8 @@ public Kernel BuildKernel(bool overwrite = false, string overrideNetworkFile = n
                .File(Path.Combine(_targetConfigFolder, _fileName),
                     rollingInterval: RollingInterval.Day,
                     outputTemplate: "{Timestamp:HH:mm:ss} [{Level:u3}] ({MachineName}/{ThreadId}) {Message} ({SourceContext}){NewLine}{Exception}")
+               .Filter.ByExcluding(Matching.FromSource("Microsoft"))
```

Comment:
same here - much less nonsense displayed (but obviously when you are debugging networking then the view is very different)

Reply:
👍 lets remove this from logs for now, logging needs to be looked at more carefully later one